gcc/testsuite/rust.test/compilable/cfg_feature.rs

```diff
@@ -0,0 +1,7 @@
+use std::env;
+
+#[cfg_attr(feature = "somefeature", attribute = "someattr")]
```

Comment:
Can you name is file `cfg_attr.rs`? `cfg` attributes don't have to be of the form `feature = "..."`. That is merely what Cargo uses. You can pass almost arbitrary values to `--cfg`. Eg `--cfg foo` will enable `#[cfg(foo)]` and `--cfg bar="baz"` will enable `#[cfg(bar = "baz")]`.

Reply:
Done in b426501